release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -1055,6 +1075,21 @@ module openconfig-interfaces {
       oc-ext:telemetry-on-change;
     }
 
+    leaf link-transitions {
+      type oc-yang:counter64;
+      description
+        "This is the number of times that the underlying link state
+        (e.g., at the optical receiver) has transitioned to/from
```

Comment:
Based on feedback from operators meeting, we should change this to be specifically "and".  This way it's clear and consistent about what is being counted.     The language could be:

"from up to down and from down to up" to be more clearly worded.

Reply:
(+@jsterne) I've reworked the descriptions to try and be a lot more explicit on the expected behaviour, please can you check if you think that this text is better.

A couple more points:

1. I've not added either of these counters to sub-interfaces because the existing carrier-transitions counter is already marked as being deprecated.

2. link-transitions is marked as oc-ext:telemetry-on-change, since being notified that the underlying interface is not completely stable may be worthy of being notified of.